### DISCUSSION TITLE
Help clean up this mess?

### DISCUSSION THREAD

**Discussion by Sullysgit** [community/community#43730] [New to GitHub]
Hey everyone,

I'm new to git and coding in general.

I have gone through multiple tutorials and test projects, and have now made a mess of my git directory. Is there any way to start fresh. 

Please advise. I'm sure there is an existing thread on this, but I haven't found it yet.

Thanks in advance!

**Comment by shree675** [comment]
@Sullysgit In case you want to initialize your local git repository freshly, you can delete your **.git** folder in your directory. This folder will not be visible by default in the file explorer. To remove it, you can use the following command:
```powershell
$ rm -rf .git
```
This will remove the git repository and you can restart your process. However, you will lose all your commits and git history in that particular directory.

**Reply by Sullysgit** [reply]
Great! Will I need to do this for all repositories individually, or will it erase all repositories within my git directory?

**Reply by shree675** [reply]
@Sullysgit Generally, we have one .git folder for one local repository. If you have multiple folders which are separate git repositories, then you will have to do it individually for each local folder.

**Reply by Sullysgit** [reply]
Perfect. Thank you!

Asking so I can understand once I get into multiple projects.
